.github/workflows/actions.yaml

```diff
@@ -0,0 +1,50 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: ["NERCDL-836-try-GitHub-actions", "master"]
+  pull_request:
+    branches: ["NERCDL-836-try-GitHub-actions", "master"]
```

Comment:
Is that plan to leave this to check that it works and then remove it when we know it does so that the other branches will pick it up?

Reply:
Yes :-)